src/VEM.jl

```diff
@@ -91,7 +90,8 @@ yt::Vector{Float64},Xt::Matrix{Float64},Xt₀::Matrix{Float64},β::Vector{Float6
             B1[:,1] = Diagonal(Sig1[:,1])*getXy('T',Xt,Z0) 
            # compute α_1
             Z =  0.5*(getXy('T',Xt,Z0)./sqrt.(ϕ)).^2
-            A0[:,1] = Π.*exp.(Z./(1.0.+ϕ/σ0[1]))./sqrt.(σ0[1]*ϕ.+1)
+            A0[:,1] = Π.*exp.(Z./(1.0.+ϕ/σ0[1]))./sqrt.(ϕ.^(-1).+1)
```

Comment:
we should work on the log scale first, then exp() it. eg instead of A/B, we do exp(log(A) - log(B))